--- patch/_stdint.h	2021-10-25 14:23:30.172489610 +0200
+++ /usr/include/newlib/sys/_stdint.h	2021-10-25 15:04:34.874675469 +0200
@@ -41,11 +41,11 @@
 
 #ifdef ___int32_t_defined
 #ifndef _INT32_T_DECLARED
-typedef __int32_t int32_t ;
+typedef int int32_t ;
 #define _INT32_T_DECLARED
 #endif
 #ifndef _UINT32_T_DECLARED
-typedef __uint32_t uint32_t ;
+typedef unsigned int uint32_t ;
 #define _UINT32_T_DECLARED
 #endif
 #define __int32_t_defined 1
