--- XR809SDK/gcc.mk	2021-10-13 18:38:09.000000000 +0200
+++ XR809SDK_patches/gcc.mk	2021-10-25 09:43:42.000000000 +0200
@@ -5,7 +5,7 @@
 # ----------------------------------------------------------------------------
 # cross compiler
 # ----------------------------------------------------------------------------
-CC_DIR := ~/tools/gcc-arm-none-eabi-4_9-2015q2/bin
+CC_DIR := /usr/bin
 CC_PREFIX := $(CC_DIR)/arm-none-eabi-
 
 AS      := $(CC_PREFIX)as
@@ -79,10 +79,10 @@
 
 CC_FLAGS = $(CPU) -c $(DBG_FLAG) -fno-common -fmessage-length=0 \
 	-fno-exceptions -ffunction-sections -fdata-sections -fomit-frame-pointer \
-	-Wall -Werror -Wpointer-arith -Wno-error=unused-function \
+	-Wall -Werror -Wpointer-arith -Wno-error=unused-function -Wno-error=misleading-indentation -Wno-error=stringop-truncation\
 	-MMD -MP $(OPTIMIZE_FLAG)
 
-LD_FLAGS = $(CPU) -Wl,--gc-sections --specs=nano.specs \
+LD_FLAGS = $(CPU) -Wl,--gc-sections --specs=nosys.specs \
 	-Wl,-Map=$(basename $@).map,--cref
 
 # config symbols
